templates/adp-backend-template-node/skeleton/${{ 'db-migration.Dockerfile' if values.app_infra_postgres_db_check == 'yes' }}

```diff
@@ -1,15 +1,7 @@
-# Use Liquibase as base image
-FROM liquibase/liquibase:4.25
+# Use ADP PostgreSql migration image as base image
+FROM {{adpSharedAcrName}}.azurecr.io/image/adp-postgres-migration:1.0.0
```

Comment:
does this affect Local development?

Reply:
No, we use this dockerfile in the build stage only.